### DISCUSSION TITLE
p/da292c72cb5e41b49a540d0e1f819634/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#48] [Announcements]
# p/da292c72cb5e41b49a540d0e1f819634/

上周末去了东京，周三定的青旅周五下午就坐新干线过去了，周五与周六玩的很开心，周日有点疲惫未能尽兴。  周五下午前往东京的电车遇到了一些问题导致延迟，一开始就预示着这趟旅行不会太顺利。  夜晚到达东京，先和推友约了顿饭，他是通过经营管理签过来的人，还把家人一起带了过来。 约饭的楼下对面就是纪伊国屋书店。  吃完饭后，吾辈开始在新宿车站附近闲逛，附近的大街小巷在夜晚真的非常符合日式的城市印象 – 光怪

https://blog.rxliuli.com/p/da292c72cb5e41b49a540d0e1f819634/

**Comment by edenni** [comment]
原来都内樱花应该开了吗，完全不知道笑

**Reply by rxliuli** [reply]
应该有一些地方已经盛开了吧

**Comment by ken622202** [comment]
东京环境真好啊，死宅羡慕（笑
还有可以的话想了解下，
要是走语言学校签证在日本找工作的话，
在不打工的前提下，每月的生活费什么的大概需要多少，
打工的前提下（大概也没法找到正儿八经的全职工作？），大概收入又能有多少，
望指教。

**Reply by rxliuli** [reply]
已滚回国内。

> 在不打工的前提下，每月的生活费什么的大概需要多少，

可能看人，就吾辈而言，房租大概 2.3k（这是京都，东京也许可以参考北京上海），水电每个月 700+（这个非常非常贵），正常外食每个月 3k 左右，其他出行之类的看个人，吾辈之前在关西 7 天 2k 多，但每天都住青旅（当然，条件其实大部分都还可以）。

> 打工的前提下（大概也没法找到正儿八经的全职工作？），大概收入又能有多少

最低 1000/h，每周最多 28h，也就是说在 5~6k 之间，可能可以覆盖生活费。

---

虽然但是，如果你在国内过的还可以，去日本会被迫消费降级，吾辈主要感受到的是 “食” 和 “行”。简单来说吃不起水果（其实很多东西都是）是真的，打不起车也是真的，具体参考上一篇 <https://blog.rxliuli.com/p/017ba335b9cd4468ad5a70f621fba49f/#%E8%A2%AB%E8%BF%AB%E6%B6%88%E8%B4%B9%E9%99%8D%E7%BA%A7>
